src/components/mdRatingBar/mdRatingBar.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div class="md-rating-bar" :class="[themeClass]" :disabled="disabled">
+    <div class="back-stars" :disabled="disabled">
+      <md-icon v-for="i in numStars" :key="i"
+               @mouseover.native="hoverStars"
+               @click.native="clickStars"
+               @mouseout.native="onMouseOut">{{ backIcon }}</md-icon>
+    </div>
+    <div class="front-stars" :style="frontStarsStyle">
+      <md-icon v-for="i in numStars"
+               @mouseover.native="hoverStars"
+               @click.native="clickStars"
+               @mouseout.native="onMouseOut">{{ frontIcon }}</md-icon>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" src="./mdRatingBar.scss"></style>
+
+<script>
+  import theme from '../../core/components/mdTheme/mixin';
+
+  let iconSize = 24;//size of each icon from rating bar in pixels
+
+  export default {
+    props: {
+      numStars: {
+        type: Number,
+        default: 5
+      },
+      disabled: Boolean,
+      value: {
+        type: Number,
+        validator: (value) => {
+          return value >= 0 && value <= 1;
+        },
+        default: 0
+      },
+      frontIcon: {
+        type: String,
+        default: 'star'
+      },
+      backIcon: {
+        type: String,
+        default: 'star'
+      }
+    },
+    mixins: [theme],
+    data: () => ({
+      rating: NaN
```

Comment:
After remove the watch I have a bad behavior on tw-way data binding. When a extern component changes the v-model prop, rating is not updated, so the front icons is not updated too.